setup/ide.md

```diff
@@ -0,0 +1,64 @@
+# Setting up your editor (IDE)
+
+Integrated Development Environment for short IDE is much more than just text editor.
```

Comment:
IDE short for Integrated Development Environment  is much more than just a text editor.